extension/src/testStatusBar.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { window, StatusBarAlignment, StatusBarItem } from "vscode";
+
+import * as Commands from "./commands";
+import { TestLevel, TestStatus, TestSuite } from "./protocols";
+
+export class TestStatusBar {

```

Comment:
This should be a singleton. Make sure it cannot be created more than once.

Reply:
nice